src/ui/common/src/components/workflows/artifact/metricsAndChecksOverview.tsx

```diff
@@ -28,18 +28,21 @@ export const MetricsOverview: React.FC<MetricsOverviewProps> = ({
 }) => {
   const metricTableEntries = {
     schema: schema,
-    data: metrics
-      .map((metricArtf) => {
-        let name = metricArtf.name;
-        if (name.endsWith('artifact') || name.endsWith('Aritfact')) {
-          name = name.slice(0, 0 - 'artifact'.length);
-        }
-        return {
-          title: name,
-          value: metricArtf.result?.content_serialized,
-        };
-      })
-      .filter((x) => !!x.value),
+    data: metrics.map((metricArtf) => {
+      let title = metricArtf.name;
+      if (title.endsWith('artifact') || title.endsWith('Aritfact')) {
```

Comment:
Typo: `title.endsWith('Aritfact')` --> `title.endsWith('Artifact')`

Reply:
good catch, it was like this before 😰